### DISCUSSION TITLE
1024 해상도에서 학습 후 2048 해상도로 fine-tuning

### DISCUSSION THREAD

**Discussion by jaehwan-AI** [boostcampaitech2/object-detection-level2-cv-06#112] [ideas]
강의 10강에서 나온 내용을 실험해보고자 합니다. (@kwakjeeyoon 지윤님께서 정리해주신 것에서 발견했습니다. 감사합니다!)

![image](https://user-images.githubusercontent.com/66052001/136527963-ef4467b0-d224-4f45-a220-81ff17a85a90.png)

위 자료와 같이 비교적 낮은 해상도에서 학습을 하고 작은 박스를 잘 잡기 위해 해상도를 높여 파인튜닝을 하는 실험을 해보겠습니다.

**Comment by JiyouSeo** [comment]
낮은 해상도에서 선학습후에 어느정도 수렴하면 해상도를 높여서 파인튜닝하신다는 말씀이시죠?
혹시 선학습때는 어느정도의 해상도로 진행하실 생각이신가요?

**Reply by jaehwan-AI** [reply]
선학습 512, 1024 -> fine-tuning 1024, 2048 이렇게 실험해보고 있습니다
